Bottom: ac37c217f36cd3ab289ab7928ad5ece32caa3941
Top:    52e29216b14131b7296ad15d24ac8355ffc64dd6
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-03-23 13:07:38 +0800

Refresh of virtio_pci-fix-out-of-bound

---

diff --git a/drivers/char/virtio_console.c b/drivers/char/virtio_console.c
index ee9a0f5..e9b7e0b 100644
--- a/drivers/char/virtio_console.c
+++ b/drivers/char/virtio_console.c
@@ -1944,7 +1944,6 @@ static int init_vqs(struct ports_device *portdev)
 			io_names[j + 1] = "output";
 		}
 	}
-
 	/* Find the queues. */
 	err = portdev->vdev->config->find_vqs(portdev->vdev, nr_queues, vqs,
 					      io_callbacks,
